zerver/models.py

```diff
@@ -4458,3 +4458,16 @@ def is_authenticated(self) -> bool:
         request.user.is_authenticated verifications.
         """
         return True
+
+
+class StreamTopic(models.Model):
+    # Settings for a Topic. Ideally this eventually won't need to be associated
+    # with a stream, but needs to be as long as there's no Topic table.
```

Comment:
I would remove the "Ideally" here -- given how we handle moving topics, it's not clear that a Topic table would be an improvement over the current string field.